REST_API_DOCS.md

```
@@ -0,0 +1,1216 @@
+```
+
+**Response (201 Created)** 
+
+```json
+{
+  "status": "success",
+  "data": {
+    "id": 1,
+    "email": "user@example.com",
+    "nickname": "user",
+	  "profileImageUrl": "https://storage.googleapis.com/my-bucket/users/profile/profile.png"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)